tests/pass/tree_borrows/tree-borrows.rs

```diff
@@ -1,4 +1,6 @@
+//@revisions: default uniq
 //@compile-flags: -Zmiri-tree-borrows
+//@[uniq]compile-flags: -Zmiri-unique-is-unique
```

Comment:
Nothing here actually uses unique, or does it?

Reply:
There are `Vec`s and `Box`es in the test.

Reply:
Fair.

Reply:
Will we get any test breakage if libs decides to remove `ptr::Unique` from `Vec`? Or would that be just okay?

Reply:
We will get test breakage but that seems fine for me, we can then just adjust the tests.